draft-sheffer-wimse-s2s-protocol.md

```diff
@@ -279,6 +279,109 @@ etc., are all valid and equivalent header field names. However, case is signific
 
 ## Option 1: DPoP-Inspired Authentication {#dpop-esque-auth}
 
+This option, inspired by the OAuth DPoP specification {{?RFC9449}}, uses a DPoP-like mechanism to authenticate
+the calling workload in the context of the request. The WIMSE Identity Token {{to-wit}} is sent in the request as
+described in {{wit-http-header}}. An additional JWT, the Workload Proof Token (WPT), is signed by the private key
+corresponding to the public key in the WIT. The WPT is sent in the `Workload-Proof-Token` header field of the request.
+A WPT contains the following:
+
+* in the JOSE header:
+    * `alg`: An identifier for an appropriate JWS asymmetric digital signature algorithm corresponding to
+     the confirmation key in the associated WIT.
+    * `typ`: the WPT is explicitly typed, as recommended in {{Section 3.11 of RFC8725}},
+     using the `application/wimse-proof+jwt` media type.
+* in the JWT claims:
+    * `iss`: The issuer of the token, which is the calling workload, represented by the same value as the `sub` claim
+     of the associated WIT.
+    * `aud`: The audience of the token contains the the HTTP target URI ({{Section 7.1 of RFC9110}}) of the request
+     to which the WPT is attached, without query or fragment parts.
```

Comment:
Why isn't `aud` the WIMSE identity of the target? Wouldn't that be more consistent?

Reply:
That, to me, begs a larger philosophical and scoping question about naming and addressing and identity, which is beyond my competence. And in that lacking of competence, I've used an existing, well-known, and widely used means of identifying the target of an HTTP request.

Reply:
Hmmph, let's discuss offline. In the meantime, please remove the redundant "the" from the sentence.

Reply:
done 06948498e55d9cdf8eccc34fc00c2c7c135f2cc0 and looking forward to a lively discussion